assets/src/scripts/monitoring-location/components/hydrograph/parameters.test.js

```diff
@@ -43,27 +70,26 @@ describe('monitoring-location/components/hydrograph/parameters module', () => {
         store = configureStore({
             hydrographParameters: {}
         });
-        drawSelectionTable(div, store, '11112222');
+        drawSelectionList(div, store, '11112222');
         expect(div.select('#select-time-series').size()).toBe(0);
     });
 
-    it('Expects the selection table to be rendered with the appropriate rows and selection', () => {
+    it('Expects the selection list to be rendered with the appropriate rows and selection', () => {
         store = configureStore(TEST_STATE);
-        drawSelectionTable(div, store, '11112222');
+        drawSelectionList(div, store, '11112222');
 
         const container = div.select('#select-time-series');
         expect(container.size()).toBe(1);
-        expect(container.select('table').size()).toBe(1);
-        expect(container.selectAll('tbody tr').size()).toBe(5);
-
-        expect(container.select('tbody input:checked').attr('value')).toEqual('72019');
+        expect(container.selectAll('.grid-row-container-row').size()).toBe(5);
+        expect(container.selectAll('.expansion-toggle').size()).toBe(8); // Note - there are two for each parameter with expansion rows
+        expect(container.select('input:checked').attr('value')).toEqual('72019');
     });
 
     it('Expects changing the selection retrieves hydrograph data', () => {
         store = configureStore(TEST_STATE);
-        drawSelectionTable(div, store, '11112222');
+        drawSelectionList(div, store, '11112222');
 
-        const rowOne = div.select('tbody tr:first-child');
+        const rowOne = div.select('#container-row-00060');
         rowOne.dispatch('click');
 
         expect(store.getState().hydrographState.selectedParameterCode).toEqual('00060');
```

Comment:
It would be great to add some tests for some of the event handlers in this code. If we don't do it now we should open a ticket to do this.

Reply:
Very true. Added some tests for the 'on clicks'.